src/ds/d500/d500-auto-calibration.cpp

```diff
@@ -9,310 +9,423 @@
-                                                                               float * angle,
-                                                                               rs2_update_progress_callback_sptr progress_callback )
-    {
-        float fx[2] = { -1.0f, -1.0f };
-        float fy[2] = { -1.0f, -1.0f };
-
-        float left_rect_sides[4] = { 0.f };
-        ds_calib_common::get_target_rect_info( left,
-                                               left_rect_sides,
-                                               fx[0],
-                                               fy[0],
-                                               50,
-                                               progress_callback );  // Report 50% progress
-
-        float right_rect_sides[4] = { 0.f };
-        ds_calib_common::get_target_rect_info( right, right_rect_sides, fx[1], fy[1], 75, progress_callback );
-
-        std::vector< uint8_t > ret;
-        const float correction_factor = 0.5f;
-
-        auto table_data = get_calibration_table(); // Table is returned without the header
-        auto full_table = _curr_calibration; // During get_calibration_table header is saved in _curr_calibration
-        full_table.insert( full_table.end(), table_data.begin(), table_data.end() );
-        auto table = reinterpret_cast< librealsense::ds::d500_coefficients_table * >( full_table.data() );
-
-        float ratio_to_apply = ds_calib_common::get_focal_length_correction_factor( left_rect_sides,
-                                                                                    right_rect_sides,
-                                                                                    fx,
-                                                                                    fy,
-                                                                                    target_w,
-                                                                                    target_h,
-                                                                                    table->baseline,
-                                                                                    *ratio,
-                                                                                    *angle );
-
-        if( adjust_both_sides )
-        {
-            float ratio_to_apply_2 = sqrtf( ratio_to_apply );
-            table->left_coefficients_table.base_instrinsics.fx /= ratio_to_apply_2;
-            table->left_coefficients_table.base_instrinsics.fy /= ratio_to_apply_2;
-            table->right_coefficients_table.base_instrinsics.fx *= ratio_to_apply_2;
-            table->right_coefficients_table.base_instrinsics.fy *= ratio_to_apply_2;
-        }
-        else
-        {
-            table->right_coefficients_table.base_instrinsics.fx *= ratio_to_apply;
-            table->right_coefficients_table.base_instrinsics.fy *= ratio_to_apply;
-        }
+d500_calibration_answer d500_auto_calibrated::get_triggered_calibration_status()
```

Comment:
Rename to `get_triggered_calibration_answer` or rename the struct to `d500_calibration_status`.